packages/core/src/render3/discovery_utils.ts

```diff
@@ -113,12 +138,40 @@ export function getRootComponents(target: {}): any[] {
  * @publicApi
  */
 export function getInjector(target: {}): Injector {
-  const context = loadContext(target);
+  const context = loadLContext(target);
   const tNode = context.lView[TVIEW].data[context.nodeIndex] as TElementNode;
-
   return new NodeInjector(tNode, context.lView);
 }
 
+/**
+ * Retrieve a set of injection tokens at a given DOM node.
+ *
+ * @param element Element for which the DOM listeners should be retrieved.
+ * @publicApi
+ */
+export function getInjectionTokens(element: Element): any[] {
+  const context = loadLContext(element as HTMLElement, false) !;
```

Comment:
```suggestion
  const context = loadLContext(element as HTMLElement, false);
```

Should be inferred because of the below if-guard.

Reply:
fixed